dali/kernels/imgproc/resample/resampling_impl_cpu.h

```diff
@@ -144,37 +288,30 @@ void ResamplHorzRow(Out *out_row, int out_width, const In *in_row, int in_width,
                     const int *in_columns, const float *coeffs, int support,
                     int first_regular_col, int last_regular_col, bool flipped) {
   int x = 0;
+  // if last_regular_col < first_regular_col, then we can only use one-sided clamp
+  // up to last_regular_col-1
+  int max_one_sided_clamp = std::min(first_regular_col, last_regular_col+1);
+
+  SIMD_horz_resample_impl<Out, In> impl;
   if (flipped) {
-    for (; x < first_regular_col && x <= last_regular_col; x++) {
-      ResampleCol<static_channels, false, true>(
```

Comment:
Do we still need the regular `ResampleCol`?

Reply:
Yes, it's used for processing SIMD tails.

Reply:
Missed that